### DISCUSSION TITLE
Slack invite link dead - is that on purpose?

### DISCUSSION THREAD

**Discussion by johnnyg** [risingwavelabs/risingwave#12691] [Q&A] [answered]
I had questions about how to optimise a query under RisingWave and saw from the docs that it was recommended to ask on slack however every invite link I can see for it seems to be dead.

Are discussions (i.e. here) meant to be the place to discuss these things now or does the link just need updating?

**Comment by yingjunwu** [comment, marked as answer]
Hi @johnnyg - Thanks for reporting this! No, it's definitely not intentional. Somehow, Slack's invite link can expire randomly, even if we set it to never expire. I believe this is a bug with Slack.

I just generated a new link: https://risingwave-community.slack.com/join/shared_invite/zt-1afreobhd-5Npy1oIpUWvDA~Od6zPxTA#/shared-invite/email. People always discuss things there! Welcome to join, and feel free to post any questions!
